tests/app/main/views/test_xss.py

```diff
@@ -0,0 +1,35 @@
+# coding=utf-8
+from __future__ import unicode_literals
+
+import mock
+from ...helpers import LoggedInApplicationTest
+from dmapiclient import HTTPError
+import urllib
+
+
+class TestXSS(LoggedInApplicationTest):
+    @mock.patch('app.main.views.services.data_api_client')
+    def test_service_not_found_flash_message_injection(self, data_api_client):
```

Comment:
I think these tests should go to the corresponding view test files/classes. That's what we've done before in similar cases too, so it should be easier to figure out if we've accidentally duplicated a test this way.